integration-tests/meteor/array_test.go

```diff
@@ -13,6 +13,8 @@ func TestArrayModification(t *testing.T) {
 
 	require.NoError(t, meteor1.Send(harness.DDPMethod("insertCall", "arrayTest.initializeFixtures")))
 
+	meteor1.ClearReceiveBuffer()
```

Comment:
This seems good - the test harness has a buffer that builds up changes, and then has a different method to check that the changes sent back from the meteor server match what we expect to have changed. 

The test follows this pattern: 
1. Establish DDP connection to meteor
2. Make sure that the system is 'quiescent' (AKA meteor isn't sending us active change updates)
3. Send some changes to the meteor server (or otherwise trigger changes in mongo)
4. Wait for the meteor server to send updates over DDP
5. Wait for the system to be quiescent again, then compare the updates to what we expect meteor would have told us.